release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2457,6 +2438,38 @@ spec:
 
 * The pipeline metrics API does not support the required `pipelinerun/taskrun` histogram values from {rh-openstack} 1.6 and beyond. Consequently, the *Metrics* tab in the *Pipeline* -> *Details* page is removed instead of displaying incorrect values. There is currently no workaround for this issue. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2074767[*BZ#2074767*])
 
+* In {product-title} {product-version}, pod security admission runs globally with privileged enforcement and restricted audit logging and API warnings. Versions of the `opm` binary released earlier than {product-title} {product-version}, do not work with the restricted profile setting. As a result, an error displays if you install a catalog source that was built with an `opm` binary released earlier than {product-title} {product-version}.
```

Comment:
The note that "pod security admission runs globally with privileged enforcement and restricted audit logging and API warnings" comes from the PSA release note above.

Reply:
But, why do we still mention the `opm` here? As I said on Slack, no matter the Catalog Source created by the new or old `opm`, both of them will encounter the PSA issue, because:

- registry-server (catalog source pod’s container, but no matter the SQL-based or Filed based, both of them have this issue)

- unpack job (nothing with the CatalogSource, but payload)

Reply:
@michaelryanpeter @jianzhangbjz:
The error here occurs because:

Previous images ( no matter the format ) built with OPM < 1.23 (please confirm the version) does not define the UserID (i.e. USER 65532:65532  ) in the Dockerfile. Therefore, they cannot work with the security context RunAsNonRoot set as true which is a requirement to comply with the Pod Security standards. 

Could we improve this one and add a better description for it? 
@anik120 ^